python/pyspark/util.py

```diff
@@ -395,6 +395,10 @@ def outer(ff: Callable) -> Callable:
 
             @functools.wraps(ff)
             def inner(*args: Any, **kwargs: Any) -> Any:
+                # Propagates the active remote spark session to the current thread.
+                from pyspark.sql.connect.session import SparkSession as SCS
```

Comment:
```suggestion
                from pyspark.sql.connect.session import SparkSession as RemoteSparkSession
```

normally `RemoteSparkSession` and `ConnectSparkSession` is used as the alias